src/template/js/sdk/Events/index.js

```diff
@@ -127,11 +127,18 @@ const once = function(...args) {
 }
 
 const listen = function(...args) {
+  init()
   const [module, event, callback] = getListenArgs(...args)
   return doListen(module, event, callback, false)
 }
 
-setListener(listen)
+const init = () => {
```

Comment:
There was a js module dependency here. setEmitter and setListener are methods used from Transport, but depending on when the modules are loaded, if Events was loaded before Transport, then this doesn't work because the methods are not defined yet. It is usually not a good idea to perform an action when the module is loaded into memory. So I have changed this to lazily set the emitter and listener.

Reply:
Cool. I struggled with the best way to do that when I wrote it. This seem more robust.